sites.py

```diff
@@ -15,6 +15,30 @@
     from an exposé page.
 """
 sites = [
+    {
+        "name": "Wohnungsbaugenossenschaft DPF eG",
+        "url": "https://www.dpfonline.de/interessenten/immobilien/",
+        "success-str": '<div class="immo-a-info">',
+# still working on extracting expose info
+#        "expose-url-pattern": r'<a href="https://www\.dpfonline\.de/(immobilien/[0-9a-zA-Z-]+)/">',
+#        "expose-details": {
+#            "title": r'<h1 class="immo-caption">(.+?)</h1>',
+#            "total_rent": r'<td>Gesamtmiete</td>\s+<td>(.+?) €</td>',
+#            "rooms": r'<td>Zimmer</td>\s+<td>(.+?)</td>',
+#            "area": r'<td>Wohnfläche</td>\s+<td>(.+?) m\s+<sup>2</sup>\s+</td>',
+#        },
+    },
+    {
+        "name": "Bau- und Siedlungsgenossenschaft Postheimstätte eG",
+        "url": "https://www.postheimstätte.de/properties/",
+        "none-str": "Es tut uns leid, es wurden keine Objekte gefunden.",
+    },
+    {
+        "name": "Wohnungsbaugenossenschaft Zentrum eG",
+        "url": "https://www.wbg-zentrum.de/wohnen/wohnungsangebot-2/wohnungsangebot/",
+        "success-str": '<div class="wpb_text_column wpb_content_element ">',
+        # no expose-url-pattern because it leads to PDF files
```

Comment:
What's the harm in that? As long as you don't have an `expose-details` section, the expose links won't be retrieved. But they will be listed in the email. So I'd like to have an `expose-url-pattern`, so I can open the PDFs from the email.

Reply:
I think it works exactly that way for "Gemeinnützige Baugenossenschaft Steglitz e.G.", no?

Reply:
Now that i looked a bit more into the inner workings of this i see the point. I am going to provide the pattern with the help of [pythex](https://pythex.org). :)